augur/dates.py

```diff
@@ -116,7 +113,10 @@ def get_numerical_date_from_value(value, fmt=None, min_max_year=None):
         # year-only date is ambiguous
         value = fmt.replace('%Y', value).replace('%m', 'XX').replace('%d', 'XX')
     if 'XX' in value:
-        ambig_date = ambiguous_date_to_date_range(value, fmt, min_max_year)
+        try:
+            ambig_date = DateDisambiguator(value, fmt=fmt, min_max_year=min_max_year).range()
```

Comment:
I like the simplification here and removal of the ambiguous date wrapper function above. While looking to see if any other public code in GitHub used this function, I found out that [TreeTime defines this same function (without the refactoring into a class)](https://github.com/neherlab/treetime/blob/e49f724edb14b708b6240bac11bfa23081d896d0/treetime/utils.py#L334). I don't know if that's helpful for us in Augur, but we can at least point people to that function, if they miss it later.

Reply:
Thanks for pointing that out! I hadn't realized that it came from TreeTime, but not surprised as some of the other date parsing logic here comes from there. Added a comment for context: 003b0c1402f0bd2d60c15b8d5dc4836292a46b68